src/Tizen.Sensor/Tizen.Sensor/BatchSensor.cs

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.Sensor
+{
+    public abstract class BatchSensor<TData> : Sensor where TData : Tizen.Sensor.BatchData
+    {
+        public BatchSensor(uint index = 0) : base(index) { }
+        public IReadOnlyList<TData> Data { get; protected set; }
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <returns></returns>
+        protected abstract IReadOnlyList<TData> ConvertBatchData();
+
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="eventsPtr"></param>
+        /// <param name="events_count"></param>
```

Comment:
Method/delegate has an error in the documentation of parameter(s): Parameter 'eventsCount' has a wrong name in the documentation: 'events_count'.